CONVENTIONS.md

```diff
@@ -290,6 +290,24 @@ Resource request review history:
 * [BZ 1920159](https://bugzilla.redhat.com/show_bug.cgi?id=1920159) --
   for tracking changes in 4.7/4.8 for single-node RAN
 
+##### Allowed use of limits
+
+There is an exception process for adding limits to payload workloads. In the following
+scenarios workloads may be given permission to set limits:
+
+* Workloads that are scale-invariant and have a fixed memory or CPU cost no matter what the scale of the cluster or workload
+  * The workload must be demonstrated to use fixed memory or CPU, and the component must have a plan to detect whether the limit causes impact to end user workloads
+  * Workloads that have a dynamic range of usage are not a good candidate for fixed limits.
+  * Memory limits must be set to 25-50% above the observed maximum in standard e2e runs to ensure additional headroom
+  * Examples:
+    * A health check sidecar container that performs a very simple request
+    * A controller that looks at a fixed set of objects that cannot vary based on cluster size, workload scale, or end user action
+
+In general, per-operator dynamic calculation of limits is discouraged, and instead workloads are expected to regulate their own consumption where possible. In cases where the workload has a very large dynamic range, dynamic sharding or scaling using standard (or per operator workload) is to be preferred over dynamic limit setting.
```

Comment:
What is "dynamic sharding"?

I think the implication from the start of this paragraph is that if we're going to set limits, they need to be static regardless of the cluster profile or topology. Is that correct? Why? Maybe something like

> Workloads are expected to regular their own resource consumption instead of using explicit limits, where possible, because this ensures optimal performance in all configurations. In cases where the workload has a very large dynamic range, dynamic sharding or scaling using standard (or per operator workload) should be used instead of limit setting because it takes advantage of the scheduler's ability to distribute work across the cluster. If an exception for the use of limits is approved by a release architect, static limits are preferred over dynamic calculations of limits because changing the limits of a Pod requires restarting a component, which disrupts service, adds load to the cluster, and risks scheduling failure.

I've guessed at some reasons, but maybe you had other things in mind. Once we work out the wording, I think this paragraph would make a good introduction to this sub-section and could move up to around line ~294.